hedera-node/src/main/java/com/hedera/services/state/logic/RecordStreaming.java

```diff
@@ -112,11 +118,45 @@ public void streamSystemRecord(final RecordStreamObject rso) {
     }
 
     private void stream(final RecordStreamObject rso) {
+        if (blockManager.shouldLogEveryTransaction()) {
+            logTransaction(rso);
+        }
         while (!nonBlockingHandoff.offer(rso)) {
             // Cannot proceed until we have handed off the record.
         }
     }
 
+    private void logTransaction(final RecordStreamObject rso) {
+        final var consTimestamp = rso.getTimestamp().toString();
+        final var blockNumber = rso.getStreamAlignment();
+        final var txId = rso.getTransactionRecord().getTransactionID();
+        final var txIdString =
+                String.format(
+                        "%d.%d.%d-%d-%d",
+                        txId.getAccountID().getShardNum(),
+                        txId.getAccountID().getRealmNum(),
+                        txId.getAccountID().getAccountNum(),
+                        txId.getTransactionValidStart().getSeconds(),
+                        txId.getTransactionValidStart().getNanos());
+        final var status = rso.getTransactionRecord().getReceipt().getStatus().toString();
+        var type = "UNRECOGNIZED";
+
+        try {
+            final var transaction = rso.getTransaction();
+            final var txBody = extractTransactionBody(transaction);
+            type = txBody.getDataCase().toString();
+        } catch (InvalidProtocolBufferException e) {
+            log.error("Couldn't get transaction body", e);
+        }
+
+        final var logString =
+                String.format(
+                        "Consensus timestamp: %s, Block number: %d, Transaction ID: %s, Transaction"
+                                + " type: %s, Status: %s",
+                        consTimestamp, blockNumber, txIdString, type, status);
+        log.info(logString);
```

Comment:
Will also be faster to use, 
```
log.info("Consensus timestamp: {}, ...", consTimestamp, ...);
```

Reply:
I intentionally split the `String` creation to not be in the log function because all log functions are by definition conditional, because their execution depends on the log level and SonarCloud was giving me a `Code Smell` because of "Concatenation in a conditional function".

Reply:
And the concatenation is unavoidable since it's a long string and running `spotlessApply` automatically splits it.

Reply:
Here we can just use log4j's built-in support for `{}` substitution tho,
```
                log.info(
                        "Consensus timestamp: {}, Block number: {}, Transaction ID: {}, Transaction"
                                + " type: {}, Status: {}",
                        consTimestamp, blockNumber, txIdString, type, status);
```

Reply:
Yes, but it's still going to throw a `Code Smell`. Should I ignore that?

Reply:
I changed it so it uses the built-in support but still left the string creation to be on the line above to avoid the code smell.